libs/blocks/notification/notification.js

```diff
@@ -385,8 +385,13 @@ function setStickyAccessibilityAttributes(el) {
   if (!section) return;
 
   const checkAndSetAttributes = () => {
-    const sticky = section.classList.contains('sticky-top') || section.classList.contains('sticky-bottom');
-    if (!sticky) return false;
+    const isSticky = section.classList.contains('sticky-top') || section.classList.contains('sticky-bottom');
+    if (!isSticky) return false;
+
+    if (el.classList.contains('no-closure')) {
+      el.classList.remove('no-closure');
```

Comment:
Is the conditional critical for the conditional nested inside that decorates the close element? Or could we shortcut these two lines to just `el.classList.remove('no-closure');`? If a token doesn't exist, nothing happens.